packages/api/bin/serve.js

```diff
@@ -324,7 +379,15 @@ async function resetTables(
 }
 
 module.exports = {
+  eraseDataStack,
+  fetchElasticDefaults,
   serveApi,
   serveDistributionApi,
-  resetTables
+  resetTables,
+  addCollections: serveUtils.addCollections,
```

Comment:
Why are we exporting all this here?

Reply:
Because I wanted to export the new functions eraseDatastack and resetTables as well as all of the new serverUtils function to be exported onto serveUtils at the api level.  Is there a better way to do that?  

Part is also because I think the organization suffers some.  I think this could be split into an express-server package and our api package.  But I didn't go that far, thought maybe the next IP sprint.

Reply:
_"The functions are exported from serveUtils, which is exported from the API module root. Why do they also need to be exported here?"_ I think I was stuck thinking that you would only use modules exported by the top level package `@cumulus/api/index.js`  It seems reasonable that I could not export them to the public, and use internal knowledge in the dashboard to do `const {serveUtils } = require('@cumulus/api/bin/serveUtils')` 
I'd also have to get some functions from `@cumulus/api/bin/serve` but that's not undoable by any stretch.

As for "_why do they need to be exported in the first place?_", I figured if the dashboard wanted to use them to see data, its possible other users would want to be able to control the seeded data as well.

Reply:
Why/how does the dashboard use these functions to seed data? I generally thought that the API would seed the data and the dashboard is just hitting the endpoints?

Reply:
Why would the cumulus API know what data the dashboard tests want to use?  The cumulus api now lets a test user seed whatever data it wants.  

Since we have a live api we need to set up and tear down each time we run a test. we do that with a plugin on the dashboard
https://github.com/nasa/cumulus-dashboard/blob/CUMULUS-1102-add-local-express-api-app/cypress/plugins/seedEverything.js#L46

Reply:
that function is then added to cypress plugin and used with cy.resetState() in the cypress integration tests.

Reply:
So I def can reach down and grab what I want from the packages without too much effort.  So I will take out these exports and just be sneaky. they should probably come out of the changelog then too.

Reply:
But I still think the Dashboard should control what data gets loaded into the databases